web/app/scripts/operation/parameters-parameter.js

```diff
@@ -16,6 +17,7 @@ angular.module('biggraph').directive('parametersParameter', function(util) {
       });
       util.deepWatch(scope, 'parameters', function(parameters) {
         scope.model = JSON.stringify(parameters);
+        scope.onBlur();
```

Comment:
Could you comment on your frontend changes in a descriptive form?

Reply:
Yes. If you look at all parameter types, they all take this `onBlur` function as a parameter. (Look in `operation-parameters.js`.) This is the mechanism that triggers saving parameter changes to the backend. It was missing for `parameters-parameter` (the workspace parameters in the anchor box).

Fixing this helps with breaking the binding loop here that we discussed earlier. But even so it's not perfect and I ended up writing directly into the model.

My plan is that once all the test conversions are done we go through all these `TODO` comments and have another stab at them.